modules/ROOT/pages/kafka/kafka-connector-reference-3.0.adoc

```diff
@@ -13,262 +13,325 @@ Release Notes: xref:release-notes::connector/kafka-connector-release-notes-mule-
 | Group Id a| String |  A unique string that identifies the consumer group this consumer belongs to. |  | x
 | Bootstrap Servers a| String |  Comma-separated host-port pairs used for establishing the initial connection to the Kafka cluster. The same as the `bootstrap.servers` value you must provide to Kafka clients (producer and consumer). |  | x
-| Additional Properties a| Object |  Additional properties such as a key-value pair that you need for your connection. |  | 
+| Additional Properties a| Object |  Additional properties, as key-value pairs, that you need for your connection. |  | 
 | Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
 |===
-
-[[kafka-consumer-config_kerberos-kafka-consumer-connection]]
+[[KafkaConsumerConfig_KerberosKafkaConsumerConnection]]
 ===== Kafka Kerberos Consumer Connection
 
+
 ====== Parameters
-[%header,cols="20s,20a,35a,20a,5a"]
+
+[%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Consumer Partitions a| Number |  The number of partitions to use for the consumer. |  1 | 
+| Consumer Partitions a| Number |  The number of partitions to use for the consumer. |  `1` | 
 | Group Id a| String |  A unique string that identifies the consumer group this consumer belongs to. |  | x
 | Bootstrap Servers a| String |  Comma-separated host-port pairs used for establishing the initial connection to the Kafka cluster. The same as the `bootstrap.servers` value you must provide to Kafka clients (producer and consumer). |  | x
-| Additional Properties a| Object |  Additional properties such as a key-value pair that you need for your connection. |  | 
+| Additional Properties a| Object |  Additional properties, as key-value pairs, that you need for your connection. |  | 
 | Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
-| Kerberos Config File a| String |  Kerberos configuration file. The essential Kerberos configuration information is the default realm and the default KDC. |  | 
-| Principal a| String |  Kerberos principal |  | x
-| Keytab a| String |  Path to keytab file associated with the `principal`. |  | x
+| Kerberos Config File | String a|  Kerberos configuration file (`krb5.conf`). The essential Kerberos configuration information is the default realm and the default KDC. This parameter is optional if the path was provided as a `java.security.krb5.conf` VM parameter, or if the configuration file is already placed in one of the locations where the Java distribution looks for the configuration, for example, the Oracle distribution searches as follows: 
```

Comment:
The last sentence has too much text and reads like a run-on I think "For example..." should be in it's own sentence.